lms/djangoapps/teams/static/teams/js/spec/views/topic_teams_spec.js

```diff
@@ -85,10 +83,10 @@ define([
             );
         });
 
-        it('does not show actions for a user already in a team', function() {
-            var options = {myTeamsCollection: TeamSpecHelpers.createMockTeams()};
+        it('does not show actions for a user already in a team in the teamset', function() {
+            var options = {collection: TeamSpecHelpers.createMockTeams()};
             var teamsView = createTopicTeamsView(options);
-            verifyActions(teamsView, {showActions: false});
+            verifyActions(teamsView);
```

Comment:
i think this is actually showing the opposite, that when a user isn't on a team, the thing will be shown